src/etl/LoadBalancer.cpp

```diff
@@ -84,6 +91,27 @@ LoadBalancer::LoadBalancer(
     std::shared_ptr<NetworkValidatedLedgersInterface> validatedLedgers,
     SourceFactory sourceFactory
 )
+    : forwardedDurationHistogram_(PrometheusService::histogramInt(
+          "lb_forwarded_duration_milliseconds_histogram",
+          Labels(),
+          kHISTOGRAM_BUCKETS,
+          "The duration of processing forwarded requests"
+      ))
```

Comment:
Do we actually need a histogram here? Maybe simply counter would be enough, because histogram is quite more heavy than a single counter. And probably we don't need to analyse distribution of response time.

Reply:
https://github.com/XRPLF/clio/issues/2070 was asking for `forwarded requests processing time`, that's why I implemented it as histograms.

@godexsoft what do you need: processing times or counter?